src/app/accounts/update-push-notification-settings.ts

```diff
@@ -0,0 +1,31 @@
+import { checkedToPushNotificationSettings } from "@domain/notifications"
```

Comment:
we have a `app/comm` folder. should probably be there
comm for `communication`. we could use `communication` if `comm` is unclear.

Reply:
`comm` is unclear to me, and I would support moving to `communication`. I could still see this mutation live under the `account` use cases because you are storing account info.